tests/unit/ExtCaseInsensitiveGeneratorTest.php

```diff
@@ -26,13 +26,13 @@ class ExtCaseInsensitiveGeneratorTest extends TestBase
     public function testGenerator()
     {
         $extensions = array();
-        $generator = new ExtCaseInsensitiveGenerator('some/FILE.php');
+        $generator = new ExtCaseInsensitiveGenerator('some'.DIRECTORY_SEPARATOR.'FILE.php');
```

Comment:
I would leave the paths as it was before. The constant decreases readability

Reply:
I will revert these.